### PR TITLE
Prevent button from submitting form … differently

### PR DESCRIPTION
**Pull request by kleinfreund**
I justed watched last Sunday’s recording of the stream (yes, I did miss it again. UGH!) and noticed a thing. This is a small follow-up to #13.

Currently, form submission of the button is prevented via JavaScript, but that’s unnecessary. All one needs to do for buttons in forms to not submit the form is add `type="button"`. For some reason, the default button type is `submit` which makes the button behave like that.

Basically, this pull request just changes how form submission is prevented. There are no functional changes as far as I’m aware.

### CODE REVIEW

**Comment by noopkat** [comment]
hi @kleinfreund !

Thank you for fixing this and allowing for the removal of some redundant JavaScript 💯 

bitmoji
